packages/suite/src/views/wallet/tokens/common/TokenList.tsx

```diff
@@ -158,6 +164,8 @@ export const TokenList = ({
     const { address: unusedAddress, path } = getUnusedAddressFromAccount(account);
     const device = useSelector(selectDevice);
     const { isLocked } = useDevice();
+    const shouldShowCopyAddressModal = useSelector(selectIsCopyAddressModalShown);
+    const shouldShowUnhideTokenModal = useSelector(selectIsUnhideTokenModalShown);
```

Comment:
Why don't name these variables with similar naming like selectors?

e.g.:
```
isCopyAddressModalShown
isUnhideTokenModalShown
```

WDYT?

Reply:
I was following what was already there for `selectIsSettingsDesktopAppPromoBannerShown` and `selectIsDashboardT3T1PromoBannerShown`.

I would keep it for now and maybe in the future refactor it but it's just a nitpick